### PR TITLE
london-class-8/shadi-safaee/karma-week-2

### PR DESCRIPTION
**Pull request by ShadiSafaee**
**Volunteers: Are you marking this coursework?** _You can find a guide on how to mark this coursework in `HOW_TO_MARK.md` in the root of this repository_

# Your Details

- Your Name: Shadi Safaee
- Your City: Winchester
- Your Slack Name: Shadi

# Homework Details

- Module: HTML & CSS
- Week: 2

# Notes

- What did you find easy? All easy

- What did you find hard? nothing

- What do you still not understand? nothing

- Any other notes? -

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,19 +1,102 @@
...
+        <div id="item">
+           <img class="icon" id="thumbnail" src="img/icon-coffee.svg" alt="coffee"/>
+           <span class="sub-title">Boost your productivity</span>
+        </div>
+        <div id="item">
+           <img class="icon" id="thumbnail" src="img/icon-refill.svg" alt="re-fill"/>
+           <span class="sub-title">Pay as you go</span>
+        </div>
+      </div>
+    </section>
```
Comment (beckyf0keefe) [inline index.html:70]: good use of the section semantic tag

File: css/style.css [hunk 2]
```diff
@@ -15,5 +14,147 @@ body {
...
+#title-main {
+  color: rgb(70, 69, 69);
+  text-align: center;
+  margin-top: 10vw;
+}
+.icon {
+  width: 15vw;
+  height: 15vw;
+}
+#thumbnail:hover {
```
Comment (beckyf0keefe) [inline css/style.css:103]: good to see you using the :hover selector, it will be really important in the future

File: css/style.css [hunk 3]
```diff
@@ -15,5 +14,147 @@ body {
...
+  border-radius: 3%;
+  width: 30%;
+  font-size: 1.5vw;
+}
+
+.article-summary {
+  text-align: center;
+}
+
+#title-main {
```
Comment (beckyf0keefe) [inline css/style.css:94]: try not to use ids in your css, if we make this a class then we can reuse elsewhere if we need to

File: css/style.css [hunk 4]
```diff
@@ -15,5 +14,147 @@ body {
...
+  margin-top: 2vw;
+  height: 3vw;
+  width: 2vw;
+}
+/* styling background image & texts on the image */
+
+.big-image {
+  width: 97%;
+  margin: 0;
+}
```
Comment (beckyf0keefe) [inline css/style.css:55]: On the image that we are recreating the hero image goes all the way across the screen, a keen eye for detail is really important to make sure that we are recreating it exactly to what the image shows

File: css/style.css [hunk 5]
```diff
@@ -15,5 +14,147 @@ body {
...
+.button1 {
+  background-color: orangered;
+  border: 2vw solid orangered;
+  border-radius: 3%;
+  width: 30%;
+  font-size: 1.5vw;
+}
+
+.article-summary {
```
Comment (beckyf0keefe) [inline css/style.css:90]: this is one way to do this but after what we learned in class last week, try to have a go at using flex-box instead to achieve the same effect

File: css/style.css [hunk 6]
```diff
@@ -15,5 +14,147 @@ body {
...
+  height: 2px;
+  border-width: 0;
+  color: rgb(196, 196, 196);
+  background-color: rgb(196, 196, 196);
+}
 
+footer {
+  display: flex;
+  justify-content: center;
+}
```
Comment (beckyf0keefe) [inline css/style.css:155]: good use of flex-box

**Review by beckyf0keefe — COMMENTED** [review]
Over all it looks like you're getting there with semantic tags, just make sure to use it for all the relevant content. The site visually was looking good so overall well done with the css, just make sure that you are using flexbox where appropriate and avoiding using ids as css selectors.
As a goal if you have time please change the hero image to a css background image and change the alt tags to be more descriptive.

**Review by ShadiSafaee — COMMENTED** [review]
Thank you Becky for all your comments, I really appreciate it. All updated based on your comments.

**Pull request closed without merging**